lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -349,21 +431,27 @@ class RenderPopover extends RenderShiftedBox {
     );
   }
 
+  @override
+  void dispose() {
```

Comment:
`dispose()` should go up above all other non-lifecycle methods, and below any given `init`, `attach`, `detach`, etc.

Also, is `dispose()` the right place to null out the clip layer? Is `detach()` not the correct place for that?

Reply:
I think `dispose` is the right place, as `RenderObject` also does this with its layer handle.